docs/installation.md

```diff
@@ -0,0 +1,77 @@
+## Prerequisites
+
+Ensure you have Python version 3.9 or above installed on your system.
```

Comment:
I think we're only testing version 3.10 right now -- do we want to add 3.9 to the tests? have you checked with 3.9?

Reply:
I'll add 3.9 to the CI